app/controllers/samples_controller.rb

```diff
@@ -50,6 +53,15 @@ class SamplesController < ApplicationController
   DEFAULT_MAX_NUM_TAXONS = 30
   MAX_PAGE_SIZE_V2 = 100
   MAX_BINS = 34
+  REPORT_INFO_CACHE_EXPIRES = 365.days
```

Comment:
Isn't this a bit long? I think 30 days should suffice.

Reply:
I'm not sure ... I was thinking about samples such as medical detectives that may not be accessed for 30 days but we still want them fast.